app-misc/mosquitto/mosquitto-1.4.15.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit systemd user toolchain-funcs python-any-r1
+
+DESCRIPTION="An Open Source MQTT v3 Broker"
+HOMEPAGE="http://mosquitto.org/"
+SRC_URI="http://mosquitto.org/files/source/${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="bridge examples +persistence +srv ssl tcpd websockets systemd"
+
+RDEPEND="tcpd? ( sys-apps/tcp-wrappers )
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	srv? ( net-dns/c-ares )
+	websockets? ( net-libs/libwebsockets )"
+
+makeopts() {
+		"CC=$(tc-getCC)"
+		"LIB_SUFFIX=${LIBDIR:3}"
+		"WITH_BRIDGE=$(usex bridge)"
+		"WITH_PERSISTENCE=$(usex persistence)"
+		"WITH_SRV=$(usex srv)"
+		"WITH_TLS=$(usex ssl)"
+		"WITH_WEBSOCKETS=$(usex websockets)"
+		"WITH_WRAP=$(usex tcpd)"
+}
+
+pkg_setup() {
+	enewgroup mosquitto
+	enewuser mosquitto -1 -1 -1 mosquitto
+}
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-1.4.10-conditional-tests.patch"
+	if use persistence; then
+		sed -i -e "s:^#autosave_interval:autosave_interval:" \
```

Comment:
`/autosave_interval/s:^#::` and like that?

Reply:
What do you mean?

Reply:
I mean to simplify the sed expression. `man sed`

Reply:
Done. See latest revision.